source/process/sg_rest_markup.rst

```diff
@@ -0,0 +1,128 @@
+=============================
+Using reStructuredText Markup
+=============================
+
+The reStructuredText specification allows for a certain degree of flexibility in markup to achieve your goals. For example, you can use any one of more than a dozen characters for section title underlines, and you have the option of using overline in addition to an underline.
+
+However, for consistency and ease of use, the Mattermost documentation should use a single convention, despite the existence of allowable alternatives.
+
+For more information about reStructuredText markup, see the `reStructuredText Markup Specification`_. Additional markup constructs are implemented by Sphinx, the documentation generator used by Mattermost. For information about the additional constructs, see `Sphinx Markup Constructs`_.
+
+Use the following markup conventions in Mattermost documentation:
+
+Page titles
+===========
+
+For page titles, use an overline and an underline. Use the = character. For example:
+
+::
+  
+  =============================
+  Using reStructuredText Markup
+  =============================
+
+Unlike Markdown, overlines and underlines in reStructuredText must be at least as long as the title text.
+
+Section titles
+==============
+
+If your document has more than one section, use the = character for the section title underline. For example:
+
+::
+  
+  Section titles
+  ==============
+
+If you do need subsections, use the the - character for the first subsection level, and the \` character for the second subsection level. For example:
+
+::
+  
+  Subsection One
+  --------------
+  
+  Subsection Two
+  ``````````````
+
+Bullet lists
+============
+
+For bullet lists and sublists, use only the - character. For example:
+
+.. code-block:: none
+  
+  - list item one
+  - list item two
+    - sublist item one
+    - sublist item two
+  - list item three
+
+Numbered lists and procedure steps
+==================================
+
+Create numbered lists and procedure steps using arabic numerals for the top-level list and lower case alpha characters for the first nested list. For example:
+
+.. code-block:: none
+  
+  1. This is item one.
+  2. This is item two.
+    
+    a. This is nested item one.
+    b. This is nested item two.
+    
+  3. This is item three.
+  
+Linking to other documents
+==========================
+
+Use relative links to reference other documents in the Mattermost doc set. Also, use either the :doc: or :ref: linking mechanisms that are provided by Sphinx.
```

Comment:
This section was confusing to me, could we get it to a state where we don't need to go to an external link?